101-vm-ws2016-visualstudio2019/azuredeploy.json

```diff
@@ -0,0 +1,201 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "defaultValue": "WinSvVisualVM",
+            "metadata": {
+                "description": "The name of you Virtual Machine."
+            }
+        },
+        "VmSize": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
can we add some higher end machines (assuming some are allowed in the sub types)?

Reply:
I add all the end machines on the Dv2-series.